### ISSUE TITLE
FadBlock not working on Youtube - Google Chrome

### ISSUE THREAD

**Issue by liftacar** [0x48piraj/fadblock#235]
Fadblock currently not working for me on Google Chrome. I don't get any "YouTube has detected an adblocker" message the ads are just playing in full and not the usual < 1 sec run time.  Anyone else having this issue or idea of a fix?

**Comment by mika5710** [comment]
The same thing happening here -- have been trying to figure out how to fix this for two days now. I tried clearing my cache and nothing.

**Comment by liftacar** [comment]
> The same thing happening here -- have been trying to figure out how to fix this for two days now. I tried clearing my cache and nothing.

Same here.  But this just popped up for me today unfortunately.  Hoping a patch/update is pushed.

**Comment by patrick1647668** [comment]
Same here.

**Comment by ckeene159** [comment]
Same thing happened to me yesterday with Firefox.  I don't know how to tell if I ever had Fadblock v2.0.  I did log in to Fadblock after v2.0 was announced, something I hadn't bothered with previously even though I'd donated.  I never saw any change, but I had set Firefox to automatically clear cache on closing.  So what it was doing, both before and after I logged in to Fadblock, is it would play the ad fast but then freeze on a "skip" option.  Yesterday it stopped playing the ads fast, so it's just like no Fadblock.  I wish I could tell 1) what version of Fadblock I have, and 2) whether I'm logged in to my Fadblock account.  I assume I am logged in as it no longer offers an option to log in.

**Comment by 0x48piraj** [comment]
I am looking into the issue, the update currently works for me but I have seen some clips where the skip function wasn't working properly. People were having to manually click the Skip button but the ads were getting skipped.

Assuming you're running the latest version as you're on Google Chrome.

**Comment by liftacar** [comment]
> I am looking into the issue, the update currently works for me but I have seen some clips where the skip function wasn't working properly. People were having to manually click the Skip button but the ads were getting skipped.
> 
> Assuming you're running the latest version as you're on Google Chrome.

I think one of the issues is people don't know if they're running the newest version or not.  Maybe lay out instructions for us all to check and if not steps to update it properly.

**Comment by kpjjf1** [comment]
Same problem here in the UK

**Comment by astrieanna** [comment]
I've had the same problem since late last week. I'm on the latest version of Chrome and of the extension, in Canada.

**Comment by ckeene159** [comment]
> I've had the same problem since late last week. I'm on the latest version of Chrome and of the extension, in Canada.

How do you know that you're on the latest version of the extension?  I can't figure out what version of Fadblock I'm on.

**Comment by astrieanna** [comment]
> How do you know that you're on the latest version of the extension? I can't figure out what version of Fadblock I'm on.

When I click the extension, then the "..." menu; the menu includes an "Update" option. After clicking, it says "Updated!", so I assume I'm on the latest version.

**Comment by ckeene159** [comment]
> > How do you know that you're on the latest version of the extension? I can't figure out what version of Fadblock I'm on.
> 
> When I click the extension, then the "..." menu; the menu includes an "Update" option. After clicking, it says "Updated!", so I assume I'm on the latest version.

In my experience it always does that.  Even when the new version hasn't yet been pushed by Firefox.

**Comment by liftacar** [comment]
***FOR THOSE WONDERING HOW TO CHECK THEIR VERSION OF FADBLOCK***

These instructions are for GOOGLE CHROME

Top right part of the browser click the 3 dots ...
Extensions > Manage extensions
Find Fadblock and to the right of it it should say 2.0 or the version you're on, but to make sure you can click DETAILS and it will have the version there.

Hope that helps.

**Comment by ckeene159** [comment]
> _**FOR THOSE WONDERING HOW TO CHECK THEIR VERSION OF FADBLOCK**_
> 
> These instructions are for GOOGLE CHROME
> 
> Top right part of the browser click the 3 dots ... Extensions > Manage extensions Find Fadblock and to the right of it it should say 2.0 or the version you're on, but to make sure you can click DETAILS and it will have the version there.
> 
> Hope that helps.

Thank you!  I'm using Firefox, my Fadblock version is 2.7, and it says it was updated on September 11.  (I then clicked "Update", got "Updated!", exited Firefox, went back in, and it still says 2.7/September 11.)

**Comment by ene0s** [comment]
Using Fadblock 2.0 on Chrome 130.0.6723.71, it is not working since a few days maybe. I'm mostly afk these days anyway.

**Comment by BrianHTC** [comment]
Same. And couldn't log in to the account for the extension, no matter what I tried, the extension always shows "not activate".

**Comment by ciscoTON** [comment]
ever since I paid for the lifetime membership, the extension hasn't blocked a single ad, they just run like normal! what the hell?

**Comment by skydug** [comment]
Ads showing up with a vengeance and have to wait till skip button appears. On V2.0, have logged out + cleared cache + disabled Adblockers.

**Comment by Hufflepuffhatesyoutubeads47** [comment]
ads are no longer being skipped for me too. I paid the license, I cleared my cache, I did all the things. Nothing has worked and I feel a little ripped off.

**Comment by skydug** [comment]
Make a choice. Either you pay monthly for premium to get no ads or give your support (and some patience) to @0x48piraj to fix it. Saying you feel ripped off doesn't help make him find the solution any quicker. It takes time to investigate what changed and come up with a way to fix it. 

I feel you. The ads are so annoying to me that I now have a disdain for the brands/products being advertised. So let's just give him some time to figure things out and trust he is doing his best despite having IRL stuffs also to deal with.  

Peace ✌🏼

**Comment by Hufflepuffhatesyoutubeads47** [comment]
> Make a choice. Either you pay monthly for premium to get no ads or give your support (and some patience) to @0x48piraj to fix it. Saying you feel ripped off doesn't help make him find the solution any quicker. It takes time to investigate what changed and come up with a way to fix it.
> 
> I feel you. The ads are so annoying to me that I now have a disdain for the brands/products being advertised. So let's just give him some time to figure things out and trust he is doing his best despite having IRL stuffs also to deal with.
> 
> Peace ✌🏼

You're right, I apologize for my previous comment, I was just upset that it stopped working, as everyone is and saying stuff like that don't fix anything.

**Comment by eric-pen** [comment]
I'm still unable to block ads, and it's been ongoing for a week now. 
Videos still require a five-second wait before I can manually select 'skip

**Comment by liftacar** [comment]
@0x48piraj  -- Hey, can we get an update on the status of FadBlock and if any updates or changes will be made?

> I am looking into the issue, the update currently works for me but I have seen some clips where the skip function wasn't working properly. People were having to manually click the Skip button but the ads were getting skipped.
> 
> Assuming you're running the latest version as you're on Google Chrome.

**Comment by StateConscious** [comment]
As much as I don't like to say this, is it time to face it that we got duped? Trusted the author, paid for the plugin and here we go..

**Comment by Hufflepuffhatesyoutubeads47** [comment]
> As much as I don't like to say this, is it time to face it that we got duped? Trusted the author, paid for the plugin and here we go..

I think we need to give him some more time. Stuff like this can't be coded and fixed in a couple weeks

**Comment by StateConscious** [comment]
It's been at least a month. If he was working on it he'd figure it out by now, (other platforms like android have no issues blocking ads on youtube), but let's be realistic he's not working on it. No published updates, no nothing.

**Comment by liftacar** [comment]
I think you're likely right.  I e-mailed him a few weeks ago and he said he was working on it and I said it would be wise for him to respond to the comments on here which he said he did, but clearly he didnt

**Comment by eric-pen** [comment]
Strangely, he's still posting other stuff on GitHub, but he just hasn't responded to our questions for a long time

**Comment by liftacar** [comment]
> Strangely, he's still posting other stuff on GitHub, but he just hasn't responded to our questions for a long time

That is strange.  I'll try e-mailing him again tonight when I get home to see what's going on.

**Comment by eric-pen** [comment]
> > 奇怪的是，他仍然在 GitHub 上發布其他內容，但他只是很久沒有回覆我們的問題
> 
> 這很奇怪。今晚我回家後會嘗試再次給他發電子郵件，看看發生了什麼事。

I really hate to say this, but since quite a few people are willing to pay to support Fadblock, I think the developer should at least update us if they're having trouble with the updates, rather than just staying silent. You could even just tell me that the program won't be updated anymore so I can give up hope. That would be better than me sitting here, waiting for an answer that might never come.

**Comment by skydug** [comment]
> It's been at least a month. If he was working on it he'd figure it out by now, (other platforms like android have no issues blocking ads on youtube), but let's be realistic he's not working on it. No published updates, no nothing.

Could you share how to apply adblocking on Android? I hope it's not some APK method. 😅

**Comment by eric-pen** [comment]
@0x48piraj So, do you really have nothing to say? It’s been almost two months! If you’re not planning to update the app anymore, just say it already.

**Comment by Hufflepuffhatesyoutubeads47** [comment]
@0x48piraj ???

**Comment by StateConscious** [comment]
> [@0x48piraj](https://github.com/0x48piraj) ???

We got played..
